### DISCUSSION TITLE
Unable to expand storage space for codespace

### DISCUSSION THREAD

**Discussion by hackedXD** [community/community#62548] [Codespaces] [answered]
### Select Topic Area

Question

### Body

When attempting to alter the machine type, only two machine types are shown. I need a higher amount of storage specifically, and can't figure out how to obtain that.

Example of available machine types:
 
![image](https://github.com/community/community/assets/63695721/28091ff2-db8f-4315-b201-8ca606c5b165)

**Comment by DSDmark** [comment, marked as answer]
Hi @hackedXD,

 check out this: https://docs.github.com/en/codespaces/customizing-your-codespace/changing-the-machine-type-for-your-codespace
and https://github.com/orgs/community/discussions/7087

**Reply by hackedXD** [reply]
Thanks!
